src/Microsoft.Restier.EntityFramework/Model/ModelMapper.cs

```diff
@@ -60,10 +61,10 @@ public ModelMapper(Type dbContextType)
             if (property != null)
             {
                 var type = property.PropertyType;
-                if (type.IsGenericType &&
-                    type.GetGenericTypeDefinition() == typeof(DbSet<>))
+                var parent = GetGenericParent(type);
```

Comment:
There is a `TypeExtensions.FindGenericType` method: https://github.com/OData/RESTier/blob/993fdc85cb62919d7f714f65bfe344884cfeee2f/src/Shared/TypeExtensions.cs#L16